### PR TITLE
Use go from backports in Wheezy, fixes #29

### LINKED ISSUE #29: Gitlab 8.0 not working on wheezy (golang to old)
Got this error:
```
go build -ldflags "-X main.Version 0.2.10-20150928.143221" -o gitlab-git-http-server
# _/var/local/gitlab/gitlab-git-http-server
./githandler.go:113: undefined: strings.TrimSuffix
./githandler.go:257: undefined: strings.TrimPrefix
```
Fix:
apt-get -t wheezy-backports install golang

This needs to go into docs.

### PR DESCRIPTION
**Pull request by scibi**

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -1,6 +1,13 @@
 ---
 
 dependencies:
+  - role: debops.apt_preferences
+    tags: [ 'depend::apt_preferences', 'type::dependency', 'depend-of::gitlab', 'type::dependency' ]
+    apt_preferences_dependent_list:
+      - package: 'golang*'
```
Comment (drybjed) [inline meta/main.yml]: This should probably be `'golang golang-*` to be more explicit about the packages you select.

File: meta/main.yml [hunk 2]
```diff
@@ -1,6 +1,13 @@
 ---
 
 dependencies:
+  - role: debops.apt_preferences
+    tags: [ 'depend::apt_preferences', 'type::dependency', 'depend-of::gitlab', 'type::dependency' ]
+    apt_preferences_dependent_list:
+      - package: 'golang-*'
```
Comment (drybjed) [inline meta/main.yml]: You also need to specify the `golang` package itself, so:
```
- package: 'golang golang-*'
```

Comment (scibi) [inline meta/main.yml]: There is no `golang` package...

    root@gitlab-test1:~# COLUMNS=80 dpkg -l go*
    Desired=Unknown/Install/Remove/Purge/Hold
    | Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
    |/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
    ||/ Name           Version      Architecture Description
    +++-==============-============-============-=================================
    un  go-compiler    <none>                    (no description available)
    ii  golang-go      2:1.3.3-1~bp amd64        Go programming language compiler
    ii  golang-go-linu 2:1.3.3-1~bp amd64        Go standard library compiled for 
    un  golang-go.tool <none>                    (no description available)
    ii  golang-src     2:1.3.3-1~bp amd64        Go programming language compiler 
    un  golang-tools   <none>                    (no description available)
    un  golang-weekly- <none>                    (no description available)
    un  golang-weekly- <none>                    (no description available)
    un  golang-weekly- <none>                    (no description available)

Comment (drybjed) [inline meta/main.yml]: There is a `golang` package when you look in all packages, and not just installed ones:
```
# apt-cache search 'golang*'
golang - Go programming language compiler - metapackage
golang-dbg - Go programming language compiler - debug files
golang-doc - Go programming language compiler - documentation
golang-go - Go programming language compiler
golang-mode - Go programming language - mode for GNU Emacs
golang-src - Go programming language compiler - source files
dh-golang - debhelper add-on for packaging software written in Go (golang)
golang-go-darwin-386 - Go standard library compiled for darwin_386
golang-go-darwin-amd64 - Go standard library compiled for darwin_amd64
golang-go-freebsd-386 - Go standard library compiled for freebsd_386
golang-go-freebsd-amd64 - Go standard library compiled for freebsd_amd64
golang-go-freebsd-arm - Go standard library compiled for freebsd_arm
golang-go-linux-386 - Go standard library compiled for linux_386
golang-go-linux-amd64 - Go standard library compiled for linux_amd64
golang-go-linux-arm - Go standard library compiled for linux_arm
golang-go-netbsd-386 - Go standard library compiled for netbsd_386
golang-go-netbsd-amd64 - Go standard library compiled for netbsd_amd64
golang-go-windows-386 - Go standard library compiled for windows_386
golang-go-windows-amd64 - Go standard library compiled for windows_amd64
```

**Pull request merged**